modules/o2e_obe_salesforce/src/AvailableTimesService.php

```diff
@@ -133,4 +143,20 @@ public function getAvailableTimes(array $options = []) {
     }
   }
 
+  /**
+   * Get the time slots on the basis of start and end date form URL.
+   */
+  public function getTimesSlots() {
+    $params = $this->request->getCurrentRequest();
+    $start_date = $params->get('start_date');
+    $end_date = $params->get('end_date');
+    if ($start_date && $end_date) {
+      $response = $this->getAvailableTimes(['start_date' => $start_date, 'end_date' => $end_date]);
+      return new JsonResponse($response);
+    }
+    else {
+      return new JsonResponse(['start date and End date are missing.']);
```

Comment:
I think we don't need to return a whole object when the input is wrong or missing.
```suggestion
      return FALSE;
```